cli.js

```diff
@@ -1,19 +1,46 @@
 #!/usr/bin/env node
 'use strict';
+const path = require('path');
 const meow = require('meow');
 const createInkApp = require('.');
 
-meow(`
+const cli = meow(
+	`
 	Options
 		--typescript		Use TypeScript React template
 
 	Usage
-		$ mkdir my-cli
-		$ cd my-cli
-	  $ create-ink-app
-`);
+		$ create-ink-app <project-directory>
 
-createInkApp().catch(error => {
-	console.error(error.stack);
-	process.exit(1);
-});
+	Examples
+		$ create-ink-app my-cli
+		$ create-ink-app .
+`,
+	{
+		flags: {
+			typescript: {
```

Comment:
I would recommend making "project-directory" a flag too. It seems safer than accessing it as index 0. 

`cli.input[0]` seems fragile if for some reason something is misaligned and at position `cli.input[1]` instead.

Reply:
(I don't feel strongly about this.)

Reply:
I retract my suggestion. Convenience is more important.